airbyte-webapp/src/components/TextWithHTML/TextWithHTML.tsx

```diff
@@ -1,30 +1,24 @@
 import React from "react";
-import sanitizeHtml from "sanitize-html";
+import DOMPurify from "dompurify";
 
 type IProps = {
   text?: string;
 };
 
-const allowedAttributes = {
-  ...sanitizeHtml.defaults.allowedAttributes,
-  a: [...sanitizeHtml.defaults.allowedAttributes["a"], "rel"],
-};
-
 const TextWithHTML: React.FC<IProps> = ({ text }) => {
   if (!text) {
     return null;
   }
 
-  const sanitizedHtmlText = sanitizeHtml(text, {
-    allowedAttributes,
-    transformTags: {
-      a: sanitizeHtml.simpleTransform("a", {
-        target: "_blank",
-        rel: "noopener noreferrer",
-      }),
-    },
+  DOMPurify.addHook("afterSanitizeAttributes", function (node) {
+    if ("rel" in node) {
```

Comment:
I am not entirely understanding why we're checking for `rel` existing in `node` here? This logic should applied on all links that are transformed whether they had a `rel` attribute or not beforehand. If this is some special hack to detect this, could we please leave an inline comment about this.